de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/nodes/JsonHandler.java

```diff
@@ -23,15 +23,16 @@
  * language to store parameters (input & ouput) in a hdf file.
  *
  */
-public abstract class HDFHandler {
+public abstract class JsonHandler {
 
   // the hdf file where all model parameters are stored
-  protected static final String HDF_FILE_NAME = "parameters.h5";  
+  protected static final String JSON_FILE_NAME = "parameters.json";
+  protected static final String JSON_PARAMETERS_NAME = "fsk_parameters";
 
   protected ScriptHandler scriptHandler;
   protected ExecutionContext exec;
   
-  public HDFHandler(ScriptHandler scriptHandler, ExecutionContext exec) {
+  public JsonHandler(ScriptHandler scriptHandler, ExecutionContext exec) {

```

Comment:
Constructors of an "abstract" class should not be declared "public" (java:S5993)

Abstract classes should not have public constructors. Constructors of abstract classes can only be called in constructors of their subclasses. So there is no point in making them public. The protected modifier should be enough.